scripts/vault_login.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# This Enables and Disables individuals OIDC token to access secrets from vault
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+
+from robottelo.errors import InvalidVaultURLForOIDC
+
+HELP_TEXT = (
+    "Vault CLI in not installed in your system, "
+    "refer link https://learn.hashicorp.com/tutorials/vault/getting-started-install to "
+    "install vault CLI as per your system spec!"
+)
+
+
+def _load_env():
+    print("\nFetching vault data from .env file and Exporting vault URL .....\n")
+    ROOT_PATH = os.path.realpath(
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), os.path.pardir)
+    )
+    with open(ROOT_PATH + '/.env') as envfile:
+        envdata = envfile.read()
+    return ROOT_PATH, envdata
+
+
+def _export_vault_url(envdata):
+    vaulturl = re.findall('VAULT_URL_FOR_DYNACONF=(.*)', envdata)[0]
+    if 'localhost:8200' in vaulturl:
+        raise InvalidVaultURLForOIDC(
+            f"\033[3;31m{vaulturl} doesnt supports OIDC login,"
+            "please change url to corp vault in env file!"
+        )
+    os.environ['VAULT_ADDR'] = vaulturl
+
+
+def _vault_command(command: str):
+    vcommand = subprocess.run(command, capture_output=True, shell=True)
+    if vcommand.returncode != 0:
+        verror = str(vcommand.stderr)
+        if 'vault: command not found' in verror:
+            print(f"\033[1;31mError! {HELP_TEXT}")
+        elif 'Error revoking token' in verror:
+            print("\033[1;32mToken is alredy revoked!")
+        elif 'Error looking up token' in verror:
+            print("\033[1;33mWarning! Vault not logged in, please run 'make vault-login'!")
+        else:
+            print(f"\033[1;31mError! {verror}")
+        sys.exit(0)
```

Comment:
0 exit code is for a successful result, I'd suggest using something else

Reply:
Replaced with 1, mistakenly kept it !